extension/maxminddb.c

```diff
@@ -107,9 +107,64 @@ static int Reader_init(PyObject *self, PyObject *args, PyObject *kwds)
 static PyObject *Reader_get(PyObject *self, PyObject *args)
 {
     char *ip_address = NULL;
+    PyObject *object = NULL;
+#if PY_MAJOR_VERSION >= 3
+    PyObject *bytes = NULL;
+#endif
 
     Reader_obj *mmdb_obj = (Reader_obj *)self;
-    if (!PyArg_ParseTuple(args, "s", &ip_address)) {
+
+    if (PyArg_ParseTuple(args, "s", &ip_address)) {
+        // pass
+    } else if (PyErr_Clear(), PyArg_ParseTuple(args, "O", &object)) {
+        PyObject* module = PyImport_ImportModule("ipaddress");
+        if (module == NULL) {
+            return NULL;
+        }
+
+        PyObject* module_dict = PyModule_GetDict(module);
+        if (module_dict == NULL) {
+            return NULL;
+        }
+
+        PyObject* ipaddress_IPv4Address = PyDict_GetItemString(
```

Comment:
Shouldn't we be using `PyObject_IsInstance` or something here?

Reply:
I'm calling `PyObject_IsInstance` below. `PyObject_IsInstance` requires a PyObject as its second argument, so I don't understand how I'm supposed to use PyObject_IsInstance without getting the `PyObject_IsInstance` class first.